miden-tx/src/executor/mod.rs

```diff
@@ -3,29 +3,32 @@ use miden_objects::{
     accounts::AccountDelta,
     assembly::ProgramAst,
     transaction::{TransactionInputs, TransactionOutputs, TransactionScript},
-    Felt, TransactionResultError, Word, WORD_SIZE,
+    Felt, Word, WORD_SIZE,
 };
 use vm_core::{Program, StackOutputs, StarkField};
 
 use super::{
-    AccountCode, AccountId, DataStore, Digest, NoteOrigin, NoteScript, PreparedTransaction,
-    RecAdviceProvider, ScriptTarget, TransactionCompiler, TransactionExecutorError,
-    TransactionHost, TransactionResult,
+    AccountCode, AccountId, DataStore, Digest, ExecutedTransaction, NoteOrigin, NoteScript,
+    PreparedTransaction, RecAdviceProvider, ScriptTarget, TransactionCompiler,
+    TransactionExecutorError, TransactionHost,
 };
 use crate::{host::EventHandler, TryFromVmResult};
 
+// TRANSACTION EXECUTOR
+// ================================================================================================
+
 /// The transaction executor is responsible for executing Miden rollup transactions.
 ///
 /// Transaction execution consists of the following steps:
 /// - Fetch the data required to execute a transaction from the [DataStore].
-/// - Compile the transaction into a program using the [TransactionComplier].
-/// - Execute the transaction program and create a [TransactionWitness].
+/// - Compile the transaction into a program using the [TransactionComplier](crate::TransactionCompiler).
```

Comment:
nit: 
```suggestion
/// - Compile the transaction into a program using the [TransactionCompiler](crate::TransactionCompiler).
```
This error also seems to appear in other places

Reply:
Thanks! Fixed this in another PR.